src/System.Data.SqlClient/ref/System.Data.SqlClient.Manual.cs

```diff
@@ -7,6 +7,25 @@
 
 namespace System.Data.SqlClient
 {
+    // Declaring members from stripped base class CollectionBase
+    public sealed partial class SqlBulkCopyColumnMappingCollection : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList
```

Comment:
I am not sure about the usage statistics. I was talking strictly in terms of backward compatibility and wanted to know your thoughts on `Capacity` property.
I know that the APIs have been removed from CoreFX to streamline the surface area and to better programming models.